Parabolic/T/1D_dirichlet/temperature_analytical.vtu

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0"?>
+<VTKFile type="UnstructuredGrid" version="0.1" byte_order="LittleEndian" header_type="UInt32" compressor="vtkZLibDataCompressor">
+  <UnstructuredGrid>
+    <Piece NumberOfPoints="61" NumberOfCells="60">
+      <PointData>
+         <DataArray type="Float64" Name="Temperature_Analytical_2months" format="ascii" RangeMin="0" RangeMax="1">
+      1.00000000e+00   7.81511190e-01   5.79099555e-01   4.05380325e-01
```

Comment:
Because of the relative error between numerical solution and analytical solution is around 1e-5. 8 to 9 digits is very enough for the comparison.